openapi.yaml

```diff
@@ -4670,22 +4676,21 @@ components:
           items:
 # {% if False %}
             allOf:
-# {% else %}{{ "\n" }}            oneOf:{% endif %}
+# {% else %}{{ "\n" }}            anyOf:{% endif %}
             - type: integer
+              nullable: true
             - format: float
               type: number
+              nullable: true
             - $ref: '#/components/schemas/TimeDuration'
-            - nullable: true
+              nullable: true
           type: array
         confidence_scores:
           description: Confidence score from 0 (random guess) to 100 (very confident).
             The same order as `metrics`. May be `null` for exact metrics.
           items:
-# {% if False %}
-            allOf:
-# {% else %}{{ "\n" }}            oneOf:{% endif %}
-            - type: integer
-            - nullable: true
```

Comment:
This template is required. The JS codegen does not support oneOf nor anyOf, so we pretend to be allOf for them while generating oneOf with jinja2